client/src/components/sources/sourceHostList.js

```diff
@@ -6,9 +6,9 @@ import { EmptyState, Grid, Icon, Modal } from 'patternfly-react';
 
 class SourceHostList extends React.Component {
```

Comment:
Note:  we could embrace a little bit more on the stateless component side with this one